source/views/menus/lib/trim-names.js

```diff
@@ -5,7 +5,17 @@ export function trimStationName(stationName: string) {
   return stationName.replace(/<strong>@(.*)<\/strong>/, '$1')
 }
 
+function removeParenTags(str: string) {
+  const parensRegex = / \([^)]*?\)$/
+  while (str !== str.replace(parensRegex, '')) {
```

Comment:
Apparently it's better to use String#match instead of RegExp#test, when it might be run over the same string multiple times: https://github.com/facebook/jest/pull/3001

So let's do this instead:

```diff
-while (str !== str.replace(parensRegex, '')) {
+while (str.match(parensRegex)) {
```